src/core/services/ripple/ripple.js

```diff
@@ -326,13 +418,14 @@ InkRippleCtrl.prototype.clearTimeout = function () {
 };
 
 InkRippleCtrl.prototype.isRippleAllowed = function () {
-  var element = this.$element[0];
+  var element = this.$$element[0];
```

Comment:
Just need to repeat this again. This is **very** confusing. `$$element` and `$element` are mostly the same name.